sdk/communication/azure-communication-phonenumbers/samples/siprouting/set_sip_trunk_sample.py

```diff
@@ -0,0 +1,34 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: set_sip_trunk_sample.py
+DESCRIPTION:
+    This sample shows the usage of SIP routing client for setting the configuration
+    of a single SIP trunk.
+
+USAGE:
+    python set_sip_trunk_sample.py
+    Set the environment variables with your own values before running the sample:
+    1) COMMUNICATION_SAMPLES_CONNECTION_STRING - the connection string in your ACS account
+    2) COMMUNICATION_SAMPLES_TRUNK - SipTrunk object to be set
+"""
+
+import os
+from azure.communication.phonenumbers.siprouting import SipRoutingClient
+
+class SetSipTrunkSample(object):
+    def __init__(self):
+        connection_string = os.getenv("COMMUNICATION_SAMPLES_CONNECTION_STRING")
+        self._client = SipRoutingClient.from_connection_string(connection_string)
+
+    def set_sip_trunk_sample(self):
+        sip_trunk = os.getenv("COMMUNICATION_SAMPLES_TRUNK")
+        self._client.set_trunk(sip_trunk)
```

Comment:
Shouldn't think take an instance of SipTrunk as input?

Reply:
Changed samples to either take environment variables, in case of simple inputs and provided example values, in case of structured inputs like list of routes or list of trunks.